docs/swarm/swarm-tutorial/add-nodes.md

```diff
@@ -19,49 +19,51 @@ to add worker nodes.
 1. Open a terminal and ssh into the machine where you want to run a worker node.
 This tutorial uses the name `worker1`.
 
-2. Run the following command to create a worker node joined to
-the existing swarm:
+2. Run the command produced by the `docker swarm init` output from the previous
+step to create a worker node joined to the existing swarm:
```

Comment:
Nit: the previous step didn't involve running `docker swarm init`. Maybe "...produced by the `docker swarm init` command that created the swarm"?